tests/test_export_onnx.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import io
+import os
+import unittest
+import torch
+
+from torch.hub import _check_module_exists
+
+from detectron2 import model_zoo
+from detectron2.config import get_cfg
+from detectron2.export import add_export_config
+from detectron2.export.flatten import TracingAdapter
+from detectron2.modeling import build_model
+from detectron2.utils.env import TORCH_VERSION
+from detectron2.utils.testing import (
+    get_sample_coco_image,
+    SLOW_PUBLIC_CPU_TEST)
+
+
+SUPPORTED_ONNX_OPSET = 14
+
+
+@unittest.skipIf(not _check_module_exists("onnx"), "ONNX not installed.")
```

Comment:
Actually, could you include https://github.com/facebookresearch/detectron2/pull/4190 into this PR so the newly added tests can actually run?